src/interpreter/type_cast.rs

```diff
@@ -0,0 +1,224 @@
+use crate::interpreter::types::{Numeric, PactType, StringLike};
+use core::any::Any;
+use core::convert::TryFrom;
+use std::string::String;
```

Comment:
Can use no_std variants here I think its under `alloc` instead of `std`

Reply:
Yep it is using `alloc` which is imported as `std` 😀 
https://github.com/cennznet/pact/blob/master/src/lib.rs#L19